lib/jsonapi/active_model_error_serializer.rb

```diff
@@ -49,8 +73,10 @@ class ActiveModelErrorSerializer < ErrorSerializer
         { pointer: "/data/attributes/#{error_key}" }
       elsif rels.include?(error_key)
         { pointer: "/data/relationships/#{error_key}" }
+      elsif error_key == :base
+        { pointer: '/data' }
       else
-        { pointer: '' }
+        { pointer: nil }
```

Comment:
@johvet let's double check if this is a valid JSONAPI response, more context here:
https://jsonapi.org/examples/#error-objects-source-usage

Reply:
Hm, I do not remember the reasons for me to change this to nil, but after reading the JSONAPI spec above, using `''` seems to be the correct way.

Reply:
I think the reason for `nil` is because the pointer is unknown/unmapped in this situation but `''` means "the root of the document", which is incorrect here